src/stores/utilities/pacbioLibraries.js

```diff
@@ -30,28 +30,23 @@ const validateLibraryFields = (library) => {
  * Formats and transforms libraries.
  *
  * @param {Object} libraries - The libraries to format and transform.
- * @param {Object} tubes - The tubes associated with the libraries.
  * @param {Object} tags - The tags associated with the libraries.
  * @param {Object} requests - The requests associated with the libraries.
  * @returns {Array<Object>} - The formatted and transformed libraries.
  */
-const formatAndTransformLibraries = (libraries, tubes, tags, requests) =>
-  Object.values(libraries)
-    .filter((library) => library.tube)
```

Comment:
I believe we filtered them because on the old architecture we had libraries from pools which wouldn't have their own barcode and we wouldn't want to show up on this page. But this should no longer be the case.